app/Console/Commands/SyncFaultTypes.php

```diff
@@ -38,31 +38,43 @@ public function __construct()
      */
     public function handle()
     {
-        $type = $this->argument('category');        
+        $type = $this->argument('category');
         switch ($type) {
             case 'computers':
                 $this->info('Updating devices with computer fault types');
                 $Faultcat = new Faultcat;
                 $result = $Faultcat->updateDevices();
                 if ($result) {
-                    $this->info($result . ' rows updated');
+                    $this->info($result . ' rows updated with fault_type');
                 } else {
-                    $this->info('0 rows updated');
+                    $this->info('0 rows updated with fault_type');
+                }
+                $result = $Faultcat->updateDevicesWithEmptyProblem();
+                if ($result) {
+                    $this->info($result . ' rows updated with Unknown');
+                } else {
+                    $this->info('0 rows updated with Unknown');
                 }
                 break;
             case 'mobiles':
                 $this->info('Updating devices with mobile fault types');
                 $Mobifix = new Mobifix;
```

Comment:
Just to note I got an error here - adding use App\Mobifix to using statements resolved it.